Soccar [UI]/src/nl/soccar/ui/rmi/ClientController.java

```diff
@@ -78,6 +63,19 @@ public static ClientController getInstance() {
         return INSTANCE;
     }
 
+    public void initialize() throws RemoteException, NotBoundException {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.rmi.RemoteException, java.rmi.NotBoundException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1160)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)